<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerGlobalSettingsWidget</class>
 <widget class="QWidget" name="ControllerGlobalSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>665</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="mainLayout" columnstretch="1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="multitapGroup">
     <property name="title">
      <string>Controller Multitap</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="multitapPort1">
        <property name="text">
         <string>Multitap on Console Port 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="multitapPort2">
        <property name="text">
         <string>Multitap on Console Port 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>The multitap enables up to 8 controllers to be connected to the console. Each multitap provides 4 ports. Multitap is not supported by all games.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="dinputGroup">
     <property name="title">
      <string>DInput Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0">
       <widget class="QCheckBox" name="enableDInputSource">
        <property name="text">
         <string>Enable DInput Input Source</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>The DInput source provides support for legacy controllers which do not support XInput. Accessing these controllers via SDL instead is recommended, but DirectInput can be used if they are not compatible with SDL.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="profileSettings">
     <property name="title">
      <string>Profile Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>When this option is enabled, hotkeys can be set in this input profile, and will be used instead of the global hotkeys. By default, hotkeys are always shared between all profiles.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="useProfileHotkeyBindings">
        <property name="text">
         <string>Use Per-Profile Hotkeys</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="sdlGroup">
     <property name="title">
      <string>SDL Input Source</string>
     </property>
     <layout class="QGridLayout" name="sdlGridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="enableSDLSource">
        <property name="text">
         <string>Enable SDL Input Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>The SDL input source supports most controllers, and provides advanced functionality for DualShock 4 / DualSense pads in Bluetooth mode (Vibration / LED Control).</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="enableSDLEnhancedMode">
          <property name="text">
           <string>DualShock 4 / DualSense Enhanced Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ledSettings">
          <property name="toolTip">
           <string>Controller LED Settings</string>
          </property>
          <property name="icon">
           <iconset theme="lightbulb-line"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="enableSDLRawInput">
        <property name="text">
         <string>Enable SDL Raw Input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="enableSDLIOKitDriver">
        <property name="text">
         <string>Enable IOKit Driver</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="enableSDLMFIDriver">
        <property name="text">
         <string>Enable MFI Driver</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mouseGroup">
     <property name="title">
      <string>Mouse/Pointer Source</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>PCSX2 allows you to use your mouse to simulate analog stick movement.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QCheckBox" name="enableMouseMapping">
          <property name="text">
           <string>Enable Mouse Mapping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mouseSettings">
          <property name="toolTip">
           <string>Controller LED Settings</string>
          </property>
          <property name="text">
           <string>Settings...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="xinputGroup">
     <property name="title">
      <string>XInput Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>The XInput source provides support for Xbox 360 / Xbox One / Xbox Series controllers, and third party controllers which implement the XInput protocol.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="enableXInputSource">
        <property name="text">
         <string>Enable XInput Input Source</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="8">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Detected Devices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="deviceList">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enableSDLSource</tabstop>
  <tabstop>enableSDLEnhancedMode</tabstop>
  <tabstop>ledSettings</tabstop>
  <tabstop>enableSDLRawInput</tabstop>
  <tabstop>enableSDLIOKitDriver</tabstop>
  <tabstop>enableSDLMFIDriver</tabstop>
  <tabstop>enableXInputSource</tabstop>
  <tabstop>enableDInputSource</tabstop>
  <tabstop>enableMouseMapping</tabstop>
  <tabstop>mouseSettings</tabstop>
  <tabstop>multitapPort1</tabstop>
  <tabstop>multitapPort2</tabstop>
  <tabstop>useProfileHotkeyBindings</tabstop>
  <tabstop>deviceList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
